README.md

```diff
@@ -16,7 +14,7 @@ var request = require('superagent'),
 
 request
     .get('/d2l/api/lp/1.5/users/whoami')
-    .use(auth)
+    .use(auth())
```

Comment:
So, yes, creating it once would be better (once per set of scopes desired). I'll update it to do it within the require statement.